rest-catalog/quarkus-rest-catalog/src/intTest/java/org/projectnessie/restcatalog/server/resources/NessieDockerTestResourceLifecycleManager.java

```diff
@@ -170,7 +170,18 @@ public NessieContainer() {
         //   since Keycloak is configured to return tokens with that specific address as the issuer
         //   claim, regardless of the client IP address.
         CustomKeycloakContainer keycloak = KeycloakTestResourceLifecycleManager.getKeycloak();
-        assert keycloak != null;
+
+        String keycloakContainerIpAddress =
+            keycloak
+                .getContainerInfo()
+                .getNetworkSettings()
+                .getNetworks()
+                .values()
+                .iterator()
+                .next()
+                .getIpAddress();
+        withExtraHost("keycloak", keycloakContainerIpAddress);
```

Comment:
Nice 👍

Reply:
Looks like two host names are there (`keycloak` and the generated one) - but only one (random? literally smaller/higher?) is present in other containers.